.gitignore

```diff
@@ -1,3 +1,4 @@
 node_modules/
 package-lock.json
 lerna-debug.log
+.DS_Store
```

Comment:
This is OS specific item that should be in the global gitignore.

Reply:
Alright, I will make that change.